include/bout_types.hxx

```diff
@@ -49,10 +49,12 @@ enum REGION {RGN_ALL, RGN_NOBNDRY, RGN_NOX, RGN_NOY, RGN_NOZ};
 const std::string& REGION_STRING(REGION region);
 
 /// To identify particular directions (in index space)
-enum class DIRECTION { X = 0, Y = 1, Z = 3, YAligned = 4, YOrthogonal = 5 };
+enum class DIRECTION { X = 0, Y = 1, Z = 3, YAligned = 4, YOrthogonal = 5, Special = 6, Null = 7 };
```

Comment:
Also I realise you've only updated this bit, but note that we don't need the explicit initialisers here, we could just have `enum class DIRECTION {X, Y, Z, ...}`

Reply:
Sounds right to me. @d7919 was there a reason for skipping `2` when initializing the values?

Reply:
I don't remember I'm afraid -- I suspect not. Probably I had something like `{X, Y, YOther, Z}` and then wanted to expand to more `Y` types and was easiest to just stick both "special" cases at the end rather than leave `YOther` where it was and add another at the end.